src/PowerFlowData.jl

```diff
@@ -104,6 +104,7 @@ struct PowerFlowData{
     neighbors::Vector{Set{Int}}
     converged::Vector{Bool}
     loss_factors::Union{Matrix{Float64}, Nothing}
+    calc_loss_factors::Bool
```

Comment:
use full words in the public interface. 

```suggestion
    calculate_loss_factors::Bool
```

Reply:
wait, the `calculate_loss_factors` should be an opt in only. Why have two ?

Reply:
yes, it is needed to suppress the loss factors calculation when the data instance was initialized with calc_loss_factors = true

Reply:
Again, I've mentioned this before that it isn't a good practice to have options on the source code that are for testing. Mixing up testing options with source code is problematic.